### PR TITLE
Sameeran/docs 14 details on the allocation id in ff

### PR DESCRIPTION
**Pull request by sameerank**

### CODE REVIEW

File: docs/how-tos/event-logging.md [hunk 1]
```diff
@@ -18,21 +18,25 @@ Eppo's SDKs include either an assignment logger base class or an interface, in w
...
+| experiment                          | subject | variation | timestamp                  | subject_attributes    |
+| :---------------------------------- | :------ | :-------- | :------------------------- | :-------------------- |
+| `recommendation-algo-allocation-17` | `6141`  | `control` | `2021-06-22T17:35:12.000Z` | `{ "country": "US" }` |
 
-It's ok for this table to contain duplicate rows for the same subject. Further, if a subject is assigned to mulitple variants, Eppo will automatically remove them from the analysis.
+It's ok for this table to contain duplicate rows for the same subject. If a subject is assigned to mulitple variants, Eppo will automatically remove them from the analysis.
+
+The following fields are optional to write to your warehouse, not required by Eppo, and provided mainly for convenience: `featureFlag` and `allocation`. When you call a function to get the assignment, the `experiment` key is generated by combining the `featureFlag` and `allocation` fields. If `allocation` is the key of an experiment allocation, then the assignment data within this experiment key is valid for experiment analysis by Eppo.
```
Comment (sameerank) [inline docs/how-tos/event-logging.md:39]: Is this easy to understand/follow?

File: docs/feature-flags/sdks/android.md [hunk 2]
```diff
@@ -67,16 +67,18 @@ EppoClient eppoClient = new EppoClient.Builder()
...
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
+| `featureFlag` (string)    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `allocation` (string)     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
 
 :::note
-More examples of logging (with Segment, Rudderstack, mParticle, and Snowplow) can be found in the [event logging](./how-tos/event-logging/) page.
```
Comment (sameerank) [inline docs/feature-flags/sdks/android.md:79]: Broken link to event-logging page fixed

File: docs/feature-flags/sdks/ios.md [hunk 3]
```diff
@@ -49,9 +49,9 @@ Task {
 }
 ```
 
-It is reccomended to wrap initialization in a `Task` block in order to perform network request asynchronously.
```
Comment (sameerank) [inline docs/feature-flags/sdks/ios.md:52]: Typo fix. Should we be concerned that our iOS SDK doesn't do event logging?

Comment (aarsilv) [reply docs/feature-flags/sdks/ios.md:52]: Like it doesn't log the assignment to the data warehouse? Yeah that sounds concerning 😛

File: docs/feature-flags/sdks/ruby.md [hunk 4]
```diff
@@ -57,16 +57,18 @@ client = EppoClient::init(config)
...
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
+| `experiment` (string)     | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17" |
+| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
```
Comment (aarsilv) [inline docs/feature-flags/sdks/ruby.md:66]: Not your code, but I _think_ in the Ruby world people generally refer to these as "Hashes" ([docs](https://ruby-doc.org/core-2.7.1/Hash.html))

File: docs/how-tos/event-logging.md (hunk 1 above)
Comment (aarsilv) [reply docs/how-tos/event-logging.md:39]: I think so (then again we've been knee deep in this for a while 😅)

**Review by aarsilv — APPROVED** [review]
👍

File: docs/feature-flags/sdks/ios.md (hunk 3 above)
Comment (schmit) [reply docs/feature-flags/sdks/ios.md:52]: I don't see any logging code in the IOS SDK

File: docs/feature-flags/sdks/android.md [hunk 5]
```diff
@@ -67,16 +67,18 @@ EppoClient eppoClient = new EppoClient.Builder()
...
+| `subject` (string)        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
+| `variation` (string)      | The experiment variation the subject was assigned to                                                                     | "control"                           |
+| `timestamp` (string)      | The time when the subject was assigned to the variation                                                                  | 2021-06-22T17:35:12.000Z            |
+| `subjectAttributes` (map) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment function | `{ "country": "US" }`               |
+| `featureFlag` (string)    | An Eppo feature flag key                                                                                                 | "recommendation-algo"               |
+| `allocation` (string)     | An Eppo allocation key                                                                                                   | "allocation-17"                     |
```
Comment (schmit) [inline docs/feature-flags/sdks/android.md:78]: Seems like we keep duplicating this, might be worthwhile to create a separate page (at some point?)

**Review by schmit — APPROVED** [review]

File: docs/feature-flags/sdks/ios.md (hunk 3 above)
Comment (sameerank) [reply docs/feature-flags/sdks/ios.md:52]: I'm going to leave event logging in iOS out of this doc update. I made a Linear ticket to get this implemented: https://linear.app/eppo/issue/FF-998/missing-logging-in-ios-sdk

**Pull request merged**